src/platform/unittest/CryptTest.cpp

```diff
@@ -347,6 +347,8 @@ TEST_F(CryptTest, HpMaskChaCha20)
         LogTestBuffer("Calculated Mask:   ", Mask, sizeof(ExpectedMask));
         FAIL();
     }
+
+    CxPlatHpKeyFree(HpKey);
```

Comment:
This is only going to free if the test passes. Likely ok for our purposes, but still something to note.

Reply:
I agree, but practically that requires a C++ abstraction for all our stuff to fix properly.